deny.toml

```
@@ -267,6 +267,6 @@ allow-git = []
 # 1 or more github.com organizations to allow git sources for
 github = [""]
```

```suggestion
github = ["lurk-lab"]
```
based on https://github.com/lurk-lab/lurk-rs/pull/1007

---

🤦